README.md

```diff
@@ -81,9 +79,8 @@ Simple example:
         identifier := fmt.Sprintf("%s_%03d", t.Format("20060102_150405"), t.Nanosecond()/1e6)
         err := knuu.InitializeWithIdentifier(identifier)
         if err != nil {
-            logrus.Fatalf("error initializing knuu: %v", err)
+           t.Fatalf("Error initializing knuu: %v:", err)
```

Comment:
`Fatalf` is not a function of t, which in this context is `time.Time`. Change it to the following to fix that:

```suggestion
           log.Fatalf("Error initializing knuu: %v:", err)
```

Reply:
Updated https://github.com/celestiaorg/knuu/pull/113/commits/0ab166810e07c81188098605a9813803729309ce

Thanks!